inception-project/inception
# #804 Easy way to resume annotation

Go back to the last location worked on to resume working from there. Make last activities clear through activities list on project dashboard page.

**What's in the PR**
* [x] resume annotation for last documents
* [X] resume curation for last documents
* depends on https://github.com/webanno/webanno/issues/1296, https://github.com/webanno/webanno/issues/1295

**How to test manually**
* Open the project dashboard page for a project you worked on.
* An activities list should be shown.
* Clicking on an activity should open the associated page and document.

**Automatic testing**

**Documentation**
